rhino.yml

```diff
@@ -0,0 +1,195 @@
+
+    # Encoder for camera-opencv(x264)
+    # x264(for research) has the better performance;
+#    video-x264-encoder-amd64:
+#        build:
+#            context: https://github.com/chalmers-revere/opendlv-video-x264-encoder.git
+#            dockerfile: Dockerfile.amd64
+#        restart: on-failure
+#        network_mode: "host"
+#        ipc: "host"
+#        volumes:
+#        - /tmp:/tmp
+#        command: "--cid=111 --name=video0.i420 --width=640 --height=480"
+
+    # Volvo Angle Sensor using lab jack
+#    dev-io-ue9-amd64:
+#        image: olbender/opendlv-device-io-ue9-amd64:v0.0.1
+#        restart: on-failure
+#        network_mode: "host"
+#        command: "opendlv-device-io-ue9 --ip=10.40.40.101 --cid=111 --verbose"
+
+    # CAN interface to access Rhino.
+    opendlv-rhino:
+        image: seresearch/opendlv-rhino-on-opendlv-on-opendlv-core-on-opendavinci-on-base:v0.1.5
+        restart: on-failure
+        network_mode: "host"
+        privileged: true
+        depends_on:
+        - odsupercomponent
+        command: "/opt/opendlv.rhino/bin/opendlv-proxy-rhino-can --freq=100 --cid=111 --id=1"
+    
+    # PS3 controller
+#    dev-ps3controller:
+#        container_name: dev-ps3controller
+#        image: chalmersrevere/opendlv-device-gamepad-multi:v0.0.10
+#        restart: on-failure
+#        network_mode: "host"
+#        devices:
+#        - "/dev/input/js0:/dev/input/js0"
+#        command: "--device=/dev/input/js0 --axis_leftright=0 --axis_updown=4 --freq=100 --acc_min=0 --acc_max=50 --dec_min=0 --dec_max=-10 --steering_min=-10 --steering_max=10 --steering_max_rate=2.0 --cid=111"
+
+    # PS4 controller 
+#    dev-ps4controller:
+#        container_name: dev-ps4controller
+#        image: chalmersrevere/opendlv-device-gamepad-multi:v0.0.10
+#        restart: on-failure
+#        network_mode: "host"
+#        devices:
+#        - "/dev/input/js0:/dev/input/js0"
+#        command: "--device=/dev/input/js0 --axis_leftright=0 --axis_updown=4 --freq=100 --acc_min=0 --acc_max=50 --dec_min=0 --dec_max=-10 --steering_min=-10 --steering_max=10 --steering_max_rate=5.0 --cid=111"
+
+######## NOTE: To make steering sluggish, reduce steering_max_rate i.e. 0 < steering_max_rate < 2  
+
+    # Web-application for viewing vehicle data.
+    vehicle-view:
+        container_name: opendlv-vehicle-view
+        image: chalmersrevere/opendlv-vehicle-view-multi:v0.0.43
+        restart: always
+        network_mode: "host"
+        volumes:
+        - ~/recordings:/opt/vehicle-view/recordings
+        - /var/run/docker.sock:/var/run/docker.sock
+        environment:
+        - OD4SESSION_CID=111
+        - OPENDLV_VEHICLE_VIEW_PLATFORM=Rhino        
+        ports:
+        - "8081:8081"
+       
+    # Web-application for viewing signals.
+    signal-viewer:

```

Comment:
I can comment it out but would prefer to keep it since with the current VV version one can't view the signal plotter and camera feed at the same time. Unless you are going to remove the microservice entirely?